tests/0120-asymmetric_subscription.c

```diff
@@ -170,9 +170,15 @@ int main_0120_asymmetric_subscription(int argc, char **argv) {
         rd_kafka_mock_topic_create(mcluster, "t4", _PART_CNT, 1);
 
 
-        do_test_asymmetric("roundrobin", bootstraps);
         do_test_asymmetric("range", bootstraps);
-        do_test_asymmetric("cooperative-sticky", bootstraps);
+
+        /* These tests would be duplicate with KIP 848 protocol,
+         * given we're not testing the assignment here,
+         * as it's mocked. */
+        if (test_consumer_group_protocol_classic()) {
+                do_test_asymmetric("roundrobin", bootstraps);
+                do_test_asymmetric("cooperative-sticky", bootstraps);
```

Comment:
We should implement `uniform` assignor on the mock broker side and test that.

Reply:
[See](https://github.com/confluentinc/librdkafka/pull/4662#discussion_r1812697875)
Here we verify the assignment given these assignors, it makes sense in classic protocol as assignors are client side code, but not with 848 where assignors are broker side code.